File: Service_eHospital_DongNai_A/Implement/NoiTru_ChamSocDichTruyenService.cs
namespace Service_eHospital_DongNai_A.Implement
{
    public class NoiTru_ChamSocDichTruyenService : BaseService<NoiTru_ChamSocDichTruyen, INoiTru_ChamSocDichTruyenRepository>
    , INoiTru_ChamSocDichTruyenService
    {
    private readonly INoiTru_ChamSocDichTruyenRepository _NoiTru_ChamSocDichTruyenRepository;
    public NoiTru_ChamSocDichTruyenService(INoiTru_ChamSocDichTruyenRepository NoiTru_ChamSocDichTruyenRepository) : base(NoiTru_ChamSocDichTruyenRepository)
    {
    _NoiTru_ChamSocDichTruyenRepository = NoiTru_ChamSocDichTruyenRepository;
    }
    }
    }

File: CRM/src/app/shared/eHospital_DongNai_A/NoiTru_ChamSocDichTruyen.service.ts
import { Injectable } from '@angular/core';
import { HttpClient, HttpParams } from '@angular/common/http';
import { environment } from 'src/environments/environment';
import { NoiTru_ChamSocDichTruyen } from './NoiTru_ChamSocDichTruyen.model';
import { BaseService } from './Base.service';
@Injectable({
    providedIn: 'root'
})
export class NoiTru_ChamSocDichTruyenService extends BaseService{
    DisplayColumns001: string[] = ['Save', 'STT', 'NoiTru_ChamSocDichTruyen_Id', 'NoiTru_ChamSoc_Id', 'Duoc_Id', 'SoLuong', 'SoMLChayNguoc', 'SoMLGiaoLai', 'NgayTao', 'NguoiTao_Id', 'NgayCapNhat', 'NguoiCapNhat_Id'];

    List: NoiTru_ChamSocDichTruyen[] | undefined;
    ListFilter: NoiTru_ChamSocDichTruyen[] | undefined;
    FormData!: NoiTru_ChamSocDichTruyen;
    constructor(public httpClient: HttpClient) {
        super(httpClient);
        this.Controller = "NoiTru_ChamSocDichTruyen";
    }
}

